src/output.zig

```diff
@@ -562,6 +562,8 @@ pub noinline fn print(comptime fmt: string, args: anytype) callconv(std.builtin.
     }
 }
 
+var lock = std.Thread.Mutex{};
```

Comment:
whats the story behind this change? where is this used?

Reply:
all `Output` instances use their own lock so it was very easy for them to clobber each other when many logs are printing at the same time on different threads

Reply:
this change should still be reverted.